src/status_im2/contexts/chat/composer/effects.cljs

```diff
@@ -112,7 +112,10 @@
      (let [edit-text        (get-in edit [:content :text])
            text-value-count (count @text-value)]
        (when (and edit @input-ref)
-         (.focus ^js @input-ref)
+         ;; A small setTimeout is necessary to ensure the statement is enqueued and will get executed
+         ;; ASAP.
+         ;; https://github.com/software-mansion/react-native-screens/issues/472
+         (js/setTimeout #(.focus ^js @input-ref) 250)
```

Comment:
We should make sure timers are cleared when the component unmounts

Reply:
Shouldn't timeouts get cleared automatically when they finish?
I thought the usage of clearTimeout was only to abort the execution of the callback we pass to setTimeout.